agent/util-scripts/pbench-tool-meister-client

```diff
@@ -1,35 +1,30 @@
 #!/usr/bin/env python3
 # -*- mode: python -*-
 
-"""pbench-tool-meister-client
+"""pbench-tool-meister-client - CLI client interface for pbench
 
-Responsible for publishing the requested tool meister action.  The
-actions can be one of "start", "stop", or "send".
+Responsible for publishing the requested Tool Meister action.
 """
 
-import json
 import logging
 import os
 import sys
 
-import redis
+from pbench.agent.constants import cli_tm_allowed_actions, redis_port
+from pbench.agent.tool_meister_client import Client
 
 
-# FIXME: move to common area
-redis_host = "localhost"
-# Port number is "One Tool" in hex 0x17001
-redis_port = 17001
-
-# FIXME: this should be moved to a shared area
-tm_channel = "tool-meister-chan"
-cl_channel = "tool-meister-client"
+def main(argv):
+    """Main program for the Tool Meister client CLI.  The command line
+    arguments are:
 
-# List of allowed actions
-allowed_actions = ("end", "init", "send", "start", "stop", "kill", "sysinfo")
+      group - the tool group on which the actions will be taken
 
+      directory - the directory where data gathered from the actions will be
+                  stored
 
-def main(argv):
-    """Main program for the tool meister client.
+      action - the particular action to take, can we one of "start", "stop",
+               or "send" (see `cli_tm_allowed_actions`).
```

Comment:
For now, we don't accept the fourth parameter, `args`, which is available via the API.  That parameter is only used by the `sysinfo` action, which is now internal and not exposed at the CLI level.